### PR TITLE
execute program from file

### PR DESCRIPTION
**Pull request by 6293**

### CODE REVIEW

**Review by dr-orlovsky — COMMENTED** [review]
Can you pls replace `unwrap`s with `expects` with a messages like "unable to open the file`?

**Comment by 6293** [comment]
> Can you pls replace unwraps with expects with a messages like "unable to open the file`?

sure. I will introduce Error struct later though

File: src/program.rs [hunk 1]
```diff
@@ -0,0 +1,81 @@
+// AluVM Runtime Environment
+// To find more on AluVM please check <https://www.aluvm.org>
+//
+// Designed & written in 2021 by
+//     Dr. Maxim Orlovsky <orlovsky@pandoracore.com>
```
Comment (dr-orlovsky) [inline src/program.rs]: I think you should put here your name, not mine

**Review by dr-orlovsky — APPROVED** [review]
ACK 73e1d5fff0208ffdf74c4ef7b0a3011ad78a72bb - but feel free to change the name in the new file.

**Review by dr-orlovsky — APPROVED** [review]
ACK https://github.com/AluVM/alure/commit/ae511b0a4f5d8a343cf49d2a21674e8b5944a07f

**Pull request merged**